addon/addon-test-support/@ember/test-helpers/build-owner.ts

```diff
@@ -2,14 +2,14 @@ import Application from '@ember/application';
 import type { Resolver } from '@ember/owner';
 
 import legacyBuildRegistry from './-internal/build-registry';
-import ContainerProxyMixin from '@ember/engine/-private/container-proxy-mixin';
-import RegistryProxyMixin from '@ember/engine/-private/registry-proxy-mixin';
+import ContainerProxyMixin from '@ember/-internals/runtime';
+import RegistryProxyMixin from '@ember/-internals/runtime';
```

Comment:
I think these are going to need to be named imports instead!

```suggestion
import {
  ContainerProxyMixin,
  RegistryProxyMixin
} from '@ember/-internals/runtime';
```